src/back/routes/notification.ts

```diff
@@ -30,6 +33,17 @@ async function sendNotification(req: WithAuth) {
     throw new RequestError('Invalid data', RequestError.BadRequest)
   }
 
+  const users = await UserModel.getDiscordIdsByAddresses(isArray(recipient) ? recipient : [recipient])
```

Comment:
![image](https://github.com/decentraland/governance/assets/45410089/f3f7a843-de29-4bfc-a1b9-862d74b63574)

In case of broadcast, it is an empty string. However, it is true that it makes no sense to call the function if it is not target (at least for now), so I'll change it

Reply:
I think we didn't think about this. What should we do with the broadcast notifications @ginocingolani? Should send those notifications to all users that has verified discord account + discord notifications activated?